plugins/nanospell/plugin.js

```diff
@@ -819,12 +818,13 @@
 				if (blockList.length > 0) {
 					startCheckOrMarkWords(getUnknownWords(combinedText), blockList);
 				}
-				editor.getSelection().selectBookmarks(bookmarks); // we may not need to select, just blow it up
+
 			}
 
 			function getWords(block) {
 				var range = editor.createRange(),
 					currentWordObj,
+					bookmarks = editor.getSelection().createBookmarks2(false),
```

Comment:
Do you think it's worth explaining the "why" of this in a comment, here? I found this explanation very useful.